app/src/main/java/com/h/pixeldroid/objects/Status.kt

```diff
@@ -115,9 +110,9 @@ data class Status(
     }
 
     fun getUsername() : CharSequence {
-        var name = account.display_name
+        var name = account.username
         if (name.isNullOrEmpty()) {
-            name = account.username
+            name = account.display_name!!
```

Comment:
Can be null, would crash the application.

Reply:
At least one of the two HAS to exist otherwise the account is broken, maybe replace `!!` by `?: "noName"`

Reply:
On devrait revoir la nullabillité de tous les champs dans tous les objects en fait... Argh